src/chain_actions/update_action.rs

```diff
@@ -60,10 +60,19 @@ impl UpdateAction {
                 create_link(time_path.hash()?, entry_address.clone(), ())?;
             }
         }
+        let updated_at = sys_time()?;
+        // get create time from the header_hash
+        let maybe_element = get::<HeaderHash>(header_hash.clone().into(), GetOptions::default())?;
```

Comment:
what's the benefit of the ::<HeaderHash> here?

Reply:
from what I remember rust analyzer was giving me feedback to specify the generic type

Reply:
figured out a better solution